frontend/resourceadm/components/ResourcePartyLists/ResourcePartyLists.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { Alert, Button, Checkbox, Heading, Link as DigdirLink } from '@digdir/design-system-react';
+import classes from './ResourcePartyLists.module.css';
+import { useGetPartyListsQuery } from 'resourceadm/hooks/queries/useGetPartyLists';
+import { StudioSpinner } from '@studio/components';
+import { useGetResourcePartyListsQuery } from 'resourceadm/hooks/queries/useGetResourcePartyLists';
+import { useAddResourcePartyListMutation } from 'resourceadm/hooks/mutations/useAddResourcePartyListMutation';
+import { useRemoveResourcePartyListMutation } from 'resourceadm/hooks/mutations/useRemoveResourcePartyListMutation';
+import { getResourcePageURL } from 'resourceadm/utils/urlUtils';
+import { NewPartyListModal } from '../NewPartyListModal/NewPartyListModal';
```

Comment:
Can this import be shortened to this?
```tsx
import { NewPartyListModal } from '../NewPartyListModal';
```
If we add an ```index.ts``` file in the ```NewPartyListModal``` folder, I belive it should be possible.